string[] mainseedlist
string[] subseedlist
string[] seed_type 
int32[] coordinates_x
int32[] coordinates_y
